src/item.cpp

```diff
@@ -2285,8 +2285,8 @@ int item::get_storage() const
     int result = (static_cast<int> (static_cast<unsigned int>( t->storage ) ) );
 
     if (item::item_tags.count("pocketed") > 0){
-        pockets = ((volume() * get_coverage()) / 100) / 1.333;
-        if (result > (volume() / 2)){
```

Comment:
This was intentional: check for gear that's already covered in pockets (cargo pants!) and reduce the gain.  Pockets on top of pockets is possible, but adds encumbrance, which this doesn't yet do.